test/suite/wttest.py

```diff
@@ -510,7 +510,7 @@ def tearDown(self, dueToRetry=False):
                     else:
                         teardown_msg += "; " + str(tmp[1])
 
-        passed = not (self.failed() or teardown_failed)
+        passed = "None" if self.skipped else not (self.failed() or teardown_failed)
```

Comment:
I like the intention to have three different values for `passed`, but I don't think it's a good idea to have a variable that can be either a string or a boolean. The use of `"None"` is particularly confusing as converting it to boolean in Python results in `True`, while converting `None` (without quotes) results in `False`.

Have you considered defining an `Enum` for `passed` or just fixing up what gets printed? Or while I'm not in favor of mixing strings and booleans, at the very least it would help if the string was changed to something else, such as `"Skipped"` and adding a block comment explaining the three options for `passed`.

Reply:
That's a good point, I agree mixing the variable type is probably a bad idea. I've changed it so that we only print passed if we haven't skipped it since it's not relevant otherwise. Let me know what you think of that.